portfolio/src/main/client/src/components/controls/theme-switcher.ts

```diff
@@ -1,26 +1,36 @@
-import '@style/index.dark.scss';
-import '@style/index.light.scss';
 import {htmlElement} from '@src/util/html';
 
-// not adding this in HTML for encapsulation
-document.body.classList.add('theme-light');
+const now = new Date();
+if (now.getHours() >= 21 || now.getHours() < 9) {
```

Comment:
Fun! You can wrap this in a setInterval/setTimeout to make this automatically change the theme at 9PM

Reply:
I feel like that would be a bit of an anti-feature since the user won't be expecting this to happen, it would be jarring. Also might just piss them off if they changed the theme themselves and it changes back unexpectedly.